From ee433cd291474ad3b995b539bf1d89cb261c5ffc Mon Sep 17 00:00:00 2001
From: Olivier Roussel <olivier.roussel@inria.fr>
Date: Tue, 14 Oct 2025 11:01:04 +0200
Subject: [PATCH] bump fmt version

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9527f50..96517b6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -184,7 +184,7 @@ endmacro(TAG_LIBRARY_VERSION)
 add_project_dependency(Eigen3 3.3.7 REQUIRED PKG_CONFIG_REQUIRES
                        "eigen3 >= 3.3.7"
 )
-add_project_dependency(fmt "9.1.0...<12" REQUIRED PKG_CONFIG_REQUIRES
+add_project_dependency(fmt "9.1.0...<13" REQUIRED PKG_CONFIG_REQUIRES
                        "fmt >= 9.1.0"
 )
 
-- 
2.43.0

